selfdrive/ui/ui.cc

```diff
@@ -177,8 +177,9 @@ static void update_state(UIState *s) {
   }
   if (sm.updated("sensorEvents")) {
     for (auto sensor : sm["sensorEvents"].getSensorEvents()) {
-      if (!Hardware::TICI() && sensor.which() == cereal::SensorEventData::LIGHT) {
-        scene.light_sensor = sensor.getLight();
+      // Use light sensor when offroad, camera exposure when onroad
+      if (!scene.started && Hardware::EON() && sensor.which() == cereal::SensorEventData::LIGHT) {
+        scene.light_sensor = std::clamp<float>(10.0 * sensor.getLight(), 0.0, 1023.0);
```

Comment:
On older releases the screen brightness was max at a sensor value of `light_sensor = 255 / 2.6 = 98`. Currently we reach max brightness at `light_sensor = 100 / 0.1 = 1000`. By multiplying by 10, the old behavior is preserved.